### PR TITLE
Replaced the 3d library

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
1) Added a null safe version of the model viewer.
2) Showing the 3d model at the mint screen instead of a new screen .

### CODE REVIEW

**Review by kiroj305 — APPROVED** [review]
Glad that you've found out the model_viewer_plus. thanks.

File: lib/easel_provider.dart [hunk 1]
```diff
@@ -291,8 +291,14 @@ class EaselProvider extends ChangeNotifier {
     return sdkResponse;
   }
 
+
```
Comment (mustafapylons) [inline lib/easel_provider.dart:294]: dead code can be removed

**Review by mustafapylons — COMMENTED** [review]
Some things to fix, otherwise code is good.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

**Pull request merged**